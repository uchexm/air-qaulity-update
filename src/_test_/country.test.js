import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Country from '../components/country';

it('Render <country /> correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <MemoryRouter>
          <Country />
        </MemoryRouter>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
