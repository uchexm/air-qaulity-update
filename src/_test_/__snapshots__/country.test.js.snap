// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render <country /> correctly 1`] = `
<li
  className="country"
>
  <svg
    className="forward"
    fill="none"
    height="1em"
    onClick={[Function]}
    stroke="currentColor"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
    style={
      Object {
        "color": undefined,
      }
    }
    viewBox="0 0 24 24"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <desc />
    <path
      d="M0 0h24v24H0z"
      fill="none"
      stroke="none"
    />
    <path
      d="M15 13l4 -4l-4 -4m4 4h-11a4 4 0 0 0 0 8h1"
    />
  </svg>
  <p
    className="countryName"
  />
  <p
    className="countryCode"
  />
</li>
`;
